### PR TITLE
update emr module to use existing security groups

### PR DESCRIPTION
**Pull request by tirumerla**
## what
* Allow EMR module to use existing managed and additional security groups created outside this module.

## why
* Make EMR module flexible for users to use their own security groups if they already have one or create new security groups if they don't have existing security group or doesn't want to use one.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -4,6 +4,67 @@ variable "zone_id" {
...
+
+variable "use_existing_additional_slave_security_group" {
+  type        = bool
+  description = "If set to `true`, will use variable `additional_slave_security_group` using an existing security group that was created outside of this module"
+  default     = false
+}
+
+variable "use_existing_service_access_security_group" {
+  type        = bool
+  description = "If set to `true`, will use variable `additional_slave_security_group` using an existing security group that was created outside of this module"
```
Comment (aknysh) [inline variables.tf:33]:
```suggestion
  description = "If set to `true`, will use variable `service_access_security_group ` using an existing security group that was created outside of this module"
```

**Review by aknysh — CHANGES_REQUESTED** [review]
see comments

File: variables.tf (hunk 1 above)
Comment (tirumerla) [reply variables.tf:33]: oops, great catch. Fix is on the way.

**Review by aknysh — APPROVED** [review]
thanks @tirumerla

**Pull request merged**